api/database/helpers/ApiEnums.php

```diff
@@ -5,44 +5,6 @@
 // TODO: any way to pull these directly from the graphql schema?
 class ApiEnums
 {
-    const OPERATIONAL_REQUIREMENT_SHIFT_WORK = 'SHIFT_WORK';
-
-    const OPERATIONAL_REQUIREMENT_ON_CALL = 'ON_CALL';
-
-    const OPERATIONAL_REQUIREMENT_TRAVEL = 'TRAVEL';
-
-    const OPERATIONAL_REQUIREMENT_TRANSPORT_EQUIPMENT = 'TRANSPORT_EQUIPMENT';
-
-    const OPERATIONAL_REQUIREMENT_DRIVERS_LICENSE = 'DRIVERS_LICENSE';
-
-    const OPERATIONAL_REQUIREMENT_OVERTIME_SCHEDULED = 'OVERTIME_SCHEDULED';
-
-    const OPERATIONAL_REQUIREMENT_OVERTIME_SHORT_NOTICE = 'OVERTIME_SHORT_NOTICE';
-
-    const OPERATIONAL_REQUIREMENT_OVERTIME_OCCASIONAL = 'OVERTIME_OCCASIONAL';
-
-    const OPERATIONAL_REQUIREMENT_OVERTIME_REGULAR = 'OVERTIME_REGULAR';
-
-    /**
-     * A collection of enums for operation_requirement in factories and seeders
-     *
-     * @return string[]
-     */
-    public static function operationalRequirements(): array
-    {
-        return [
-            self::OPERATIONAL_REQUIREMENT_SHIFT_WORK,
-            self::OPERATIONAL_REQUIREMENT_ON_CALL,
-            self::OPERATIONAL_REQUIREMENT_TRAVEL,
-            self::OPERATIONAL_REQUIREMENT_TRANSPORT_EQUIPMENT,
-            self::OPERATIONAL_REQUIREMENT_DRIVERS_LICENSE,
-            self::OPERATIONAL_REQUIREMENT_OVERTIME_SCHEDULED,
-            self::OPERATIONAL_REQUIREMENT_OVERTIME_SHORT_NOTICE,
-            self::OPERATIONAL_REQUIREMENT_OVERTIME_OCCASIONAL,
-            self::OPERATIONAL_REQUIREMENT_OVERTIME_REGULAR,
-        ];
-    }
-
     const LANGUAGE_ABILITY_ENGLISH = 'ENGLISH';
```

Comment:
Wow, there's quite a bit left in here.  Out of scope for this PR but we should probably review these and move them to a "deprecated enums" file or something.